src/Redux/API/api.js

```diff
@@ -0,0 +1,5 @@
+// import axios from 'axios';
+
+// export default axios.create({
+//     baseURL: 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/vK6OLioSURRYt4btr4gO',
+// });
```

Comment:
- [ ] Kindly remove this file from the remote repository or add it to the `.gitignore` file to make your code clean.